cek.go

```diff
@@ -0,0 +1,29 @@
+package mssql
+
+type cekTable struct {
+	entries []cekTableEntry
+}
+
+type encryptionKeyInfo struct {
+	encryptedKey  []byte
+	databaseID    int
+	cekID         int
+	cekVersion    int
+	cekMdVersion  []byte
+	keyPath       string
+	keyStoreName  string
+	algorithmName string
+}
+
+type cekTableEntry struct {
+	databaseID int
+	keyId      int
```

Comment:
KeyID

Reply:
Why should this be exported?

Reply:
Oh nevermind, you were referring to the ID part 😅